### ISSUE TITLE
Default color scheme not as expected and unable to change

### ISSUE THREAD

**Issue by oaklandgit** [andmarti1424/sc-im#702]
Hi,
The default color scheme looks like this:
<img width="686" alt="image" src="https://user-images.githubusercontent.com/421615/161878486-5ce41d3c-9fd0-47c4-baa6-0f62380c1ced.png">

which seems incorrect, as help looks like:
<img width="686" alt="image" src="https://user-images.githubusercontent.com/421615/161878532-15640982-4bc8-4473-accc-d43e2411172b.png">

I tried changing the scheme with:
`:color "type=GRID_EVEN bg=BLACK fg=WHITE"` to no avail.

I may be using the command incorrectly, but regardless, this default scheme seems to be not as intended?

**Comment by andmarti1424** [comment]
That's strange. Please try replacing WHITE with DEFAULT_COLOR.. Please check syntax with one of the files inside themes directory..

**Comment by andmarti1424** [comment]
@oaklandgit, Could you check this?

**Comment by oaklandgit** [comment]
Hi @andmarti1424 I tried and there was no effect. I'll keep investigating on my end.

**Comment by andmarti1424** [comment]
That's weird. Could you please try with another terminal..?

**Comment by 1dancook** [comment]
Possibly related #627

**Comment by oaklandgit** [comment]
Yes @1dancook. My issue is the same as #627.

I also only have the problem when building from source. The homebrew version looks fine.

**Comment by andmarti1424** [comment]
@oaklandgit do you use the same virtual terminal that @1dancook uses?

**Comment by oaklandgit** [comment]
Yes. My original screenshot is from Kitty. And I also see blinking in Mac OS Terminal:

Uploading Screen Recording 2022-04-15 at 6.34.11 PM.mov…

**Comment by 1dancook** [comment]
I basically concluded that this had nothing to do with my terminal but it was because of how Scim was being built. I don't know enough to diagnose it though. But I concluded it was also nothing to do with scim itself. I think when brew builds it, brew has the right packages for it. When you build it yourself, something isn't quite right.

If someone could try to build scim on MacOS that doesn't have homebrew installed, it'd be interesting to see if there is a difference. 

That said...

I never got around to it yet myself, but it would be possible to make a brew formula to build from the dev branch and then we would be able to use the bleeding edge. I'm particularly busy in this season so I don't think I will get to it yet.

**Comment by andmarti1424** [comment]
Perhaps it has to do with the version of ncurses used for the building
